faqs/galaxy/datasets_job_status.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Understanding job statuses
+description: Job statuses will help you understand the stage of your working.
+area: datasets
+layout: faq
+box_type: tip
+contributors: [garimavs, jennaj]
+---
+
+The following job statuses will help you better understand the working stage of the process.
+
+- **Green**: The job was completed successfully.
+- **Yellow**: The job is executing. Allow this to complete! Should they run longer, they will fail with a "wall-time" error and turn _red_.
+- **Grey**: The job is being evaluated to run (new dataset) or is queued. Allow this to complete.
+- **Red**: The job has failed.
+- **Light Blue**: The job is paused. This indicates either an input has a problem or that you have [exceeded the disk quota](https://galaxyproject.org/main/#user-data-and-job-quotas) set by the administrator of the Galaxy instance you are working on.
```

Comment:
This link is very specific to the usegalaxy.org server. I would say either simply remove it, or we create a separate FAQ in the GTN that discusses disk quota in a more general, not server-specific way.

Reply:
Hey @shiltemann, I added the link because it was there in FAQ, moreover it provides users direct access to the mentioned problem. 

Can we consider adding it under the FAQ that talks on disk space, and then link it here? 
For present, I will remove the link from here.

Reply:
I don't think we currently have an FAQ about quota, but as soon as we make one we can add a link back to this one. For now just removing the link is good enough for this PR :+1: